sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/data_sources/ros_bag_data_source.py

```diff
@@ -132,6 +132,8 @@ def send_data(self, topic, data):
         msg_type = get_message(self.type_map[topic])
         msg = deserialize_message(data, msg_type)
 
+        # cSpell:enableCompoundWords
+        # cSpell:ignore imgmsg
```

Comment:
Is `enableCompoundWords` working?
why is `# cSpell:ignore imgmsg` needed?

Reply:
enableCompoundWord does not work for imgmsg (in this case it is still needed for another word (frombuffer))